integration_tests/e2e/checkYourAnswersFull.cy.ts

```diff
@@ -221,5 +228,9 @@ context('Check your answers - Full flow', () => {
       "Is there anything that Daniel Craig feels may affect their ability to work after they're released?",
     )
     abilityToWorkPage.backLink().click()
+
+    checkYourAnswersPage = Page.verifyOnPage(CheckYourAnswersPage)
```

Comment:
Similar to my comment above, this test (check your answers) only got as far as testing the user had arrived on the Check Your Answers page (ie. a `GET`), and did not attempt to submit the page.

I've changed it so it does submit the page, and asserts the user arrives on the PLP page

As above, also note this is the Create journey only, not an Update